src/lib/features.js

```diff
@@ -361,6 +361,21 @@ var featureInfos =
             return available;
         },
     },
+    FORMS:
+    {
+        description:
+        'Existence of the object document.forms with string representation "[object ' +
+        'HTMLCollection]".',
+        check:
+        function ()
+        {
+            var available =
+            typeof document === 'object' && document.forms + '' === '[object HTMLCollection]';
+            return available;
+        },
+        includes:   ['ANY_DOCUMENT'],
+        attributes: { 'web-worker': 'web-worker-restriction' },
```

Comment:
I've changed my mind. Let's just use `ANY_DOCUMENT` for all `document` related features, but let us keep `FORMS` as an alias for `ANY_DOCUMENT` when `document.forms` is required in a definition.

```suggestion
        aliasFor: 'ANY_DOCUMENT',
```

The other changes in this file can be reverted.